spring-cloud-gcp-docs/src/main/asciidoc/security-iap.adoc

```diff
@@ -35,6 +35,23 @@ dependencies {
 }
 ----
 
+== Customizable Properties
+The following properties are available with `spring.cloud.gcp.security.iap` prefix:
 
+|===
+|Name |Value |Description
```

Comment:
We normally have four columns: Name, Description, Required, Default value.

Reply:
What happens if it's required, but a default is provided in autoconfig (so, for end user, overriding is not required)?
Also, should I enforce required non-null with JSR303 annotations or something?